src/pages/Account/index.jsx

```diff
@@ -51,6 +52,7 @@ const Account = () => {
             }
             
             
+            <Footer />
```

Comment:
Isn't the `Footer` component supposed to be in the root component of the application?
Your implementation means we need to have a `Footer` component written for each pages.

Reply:
And the root component will be???

Reply:
The best place to place the `Alert` is the `Navbar` component and not the `Footer`, imaging a page that's very long, how does the user see the `ALert` component?

Reply:
The alert is going to be absolute... always floating at the bottom of the screen.